From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 6 Nov 2018 16:43:22 -0800
Subject: [PATCH] Use system mkldnn

[Update for 2019-08-01]: rebased against 1.5.0 release
[Update for 2020-06-23]: rebased against 1.6.0 release

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12bc195..02820f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -288,10 +288,6 @@ if(USE_MKLDNN)
   set(MKLDNN_USE_MKL NONE CACHE INTERNAL "" FORCE)
   set(MKLDNN_ENABLE_JIT_PROFILING OFF CACHE INTERNAL "" FORCE)
 
-  add_subdirectory(3rdparty/mkldnn)
-
-  include_directories(3rdparty/mkldnn/include)
-  include_directories(${PROJECT_BINARY_DIR}/3rdparty/mkldnn/include)
   add_definitions(-DMXNET_USE_MKLDNN=1)
   list(APPEND mxnet_LINKER_LIBS mkldnn)
 endif()
